config/application.rb

```diff
@@ -10,6 +10,7 @@ module AnacapaGithubLinker
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
+    config.autoload_paths += Dir[Rails.root.join("app", "jobs", "{*/}")]
```

Comment:
Ouch. Was this because of the use of subfolders in jobs?

Reply:
Yeah, for whatever reason the subfolders aren't automatically loaded in so I had to add that